wolfcrypt/src/ecc.c

```diff
@@ -11210,16 +11214,28 @@ int wc_ecc_encrypt(ecc_key* privKey, ecc_key* pubKey, const byte* msg,
 {
     int          ret = 0;
     word32       blockSz;
+#ifndef WOLFSSL_ECIES_OLD
+    byte         iv[16];
```

Comment:
Can we avoid hard-coded numbers? How about `AES_IV_SIZE`?

Reply:
Added MAX_IV_SIZE to ecc.h and used in array definition.